scripts/utils.py

```diff
@@ -283,3 +283,46 @@ def isGitRepo(path='./'):
     return runJob('git rev-parse --git-dir > /dev/null 2>&1', cwd=path,
                   show_command=False, show_output=False)
 
+
+def listDeprecatedFiles():
```

Comment:
I'm not a huge fan of this approach. Now you need to keep another list up-to-date.
How about detecting version of xmipp stored in the `src/xmipp/bin`, and when it doesn't match the current version, purge the folder?
You might still have issues during the development cycle, but it would work fine for releases (i.e. end users)

Reply:
Other option might be to mimic the behavior of the build system - in Python, go through all `src/xmipp/applications/programs`, and then from the `src/xmipp/bin` remove those binaries that don't have source files

Reply:
But before anything, we should just measure how big performance hit would be to simply get rid of the `src/xmipp/bin` every time

Reply:
let me check the times

Reply:
There is no performance hit deleting buld/bin and src/xmipp/bin.

Reply:
lets remove the bin folder anycase..